docs/topics/eclipse-installing-plugin.adoc

```diff
@@ -15,11 +15,11 @@ ifdef::disconnected[]
 You can install the {PluginName} in a disconnected network environment.
 endif::[]
 
-The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2020-09 R and Red Hat CodeReady Studio 12.15.
+The {PluginName} has been tested with the Eclipse IDE for Java Enterprise Developers 2021-03 and Red Hat CodeReady Studio 12.19.1.
 
 .Prerequisites
 
-* link:{CodeReadyStudioDownloadPageURL}[Red Hat CodeReady Studio] _or_ link:https://www.eclipse.org/downloads/packages/release/2020-09/r/eclipse-ide-enterprise-java-developers[Eclipse IDE for Java Enterprise Developers 2020-09 R]
+* link:{CodeReadyStudioDownloadPageURL}[Red Hat CodeReady Studio] _or_ link:https://www.eclipse.org/downloads/packages/release/2021-03/r/eclipse-ide-java-developers[Eclipse IDE for Java Enterprise Developers 2021-03].
```

Comment:
I am concerned that the user might download the Eclipse 2021-06 version by mistake because those download links also appear and the buttons are much larger than the 2021-03 download links. I suggest adding a reminder to download that specific version. Just saying that it has been tested on 2021-03 might not be a strong enough message.

Reply:
This link goes to a page that only refers to 2021-03

https://www.eclipse.org/downloads/packages/release/2021-03/r/eclipse-ide-java-developers